tools/codeowners-utils/Azure.Sdk.Tools.CodeownersLinter.Tests/CodeownersTestFiles/Editor/CodeownersHelperTests.cs

```diff
@@ -0,0 +1,535 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System;
+
+using NUnit.Framework;
+
+using Azure.Sdk.Tools.CodeownersUtils.Parsing;
+using Azure.Sdk.Tools.CodeownersUtils.Editing;
+
+namespace Azure.Sdk.Tools.Cli.Tests.Helpers
+
+{
+    [TestFixture]
+    internal class CodeownersHelperTests
+    {
+        private CodeownersEditor codeownersEditor;
```

Comment:
Sharing this instance among tests couples the tests together. That is, if any test function B relies on state set in test function A then you can get unpredictable behaviors when, for example, you run all tests in parallel. 

It's safer for each test to create its own instance of `CodeownersEditor` and initialize it from the private `codeownersContent`.